playbook.md

```diff
@@ -1630,6 +1630,110 @@ If you think this might be for you, please apply through our jobs page.
+Individuals should be proactive in taking responsibility for their own development, but line managers will provide advice and support to help team members meet their agreed short and long term goals.
+
+HR support line managers with advice and guidance on recruitment, onboarding, probation, performance, sickness and any other specialist knowledge.
+
+You can find out who your line manager is in BreatheHR on the summary section of your profile page.
+
+##### Line management principles  
+
+###### Communicate clearly
+Communication is a two-way process and it is important to check understanding and confidence, and value the other person in decision-making. Explaining the task and its objectives clearly means people know what’s expected of them and the part they play. It empowers them to take responsibility and own how they approach the work.
+
+###### Recognise achievement and celebrate success
+A simple ‘thank you’ shows someone that what they have done is appreciated, and is hugely motivating. Celebrating major, and minor, achievements is an important part of our culture at dxw. It builds team spirit and morale.
+
+###### Guard our values
+Lead by example. The actions and behaviours of line managers must always demonstrate our dxw values. We should always hold ourselves accountable to our values, and encourage the same in all of our colleagues, including those we line manage.
+
+###### Continuous constructive feedback
+Find out how each of your team members prefers to receive feedback, and develop an approach which reflects that.  Ongoing positive and constructive feedback is essential if someone is to feel valued, learn from their experiences and become better at what they do.
+
+###### Be honest
+Being honest means telling the truth and being consistent in what you say and who you say it to. Telling things as they are, while being sensitive to the situation and individual, ensures that people know where they stand and will help to build mutual trust and respect.
+
+###### Wellbeing
+Line managers look out for the health and wellbeing of the team. They are responsible for carrying out dxw's duty of care to its people.
+
+#### Directors & heads-of
+
+All members of the Directors and heads-of team are here to help answer any questions you might have. Your own Director and/or Head-of profession or location is there to provide support on matters relating to dxw and your career with us, as well as profession specific questions. Similarly, they are here if you need to escalate any job-related concerns.
+
+#### Our MD, Dave
+
+At dxw, all doors are open if you need to raise a concern or just have a chat - this includes our MD. Dave is always available to discuss any questions you have about the company and its future. Equally, if you feel you need to escalate anything to him, don’t hesitate to get in touch. Dave's diary is open and you can see it on Google Calendar. If you would like to schedule some time with him, please feel free to book something.
```

Comment:
It might be good to spell out that "MD" stands for "Managing Director" (and not, e.g. "Medical Doctor"). Not everyone will come from a background where that acronym is automatically expanded for them.

Reply:
Cool - I'll amend